cg/server/dto/orders/orders_response.py

```diff
@@ -15,3 +15,4 @@ class Order(BaseModel):
 
 class OrdersResponse(BaseModel):
     orders: list[Order]
+    total_count: int
```

Comment:
Is this only to be able to calculate the number of pages in the frontend?

Reply:
Yes! We need the total count of all matching records after filtering